Doc/library/venv.rst

```diff
@@ -81,21 +83,6 @@ It also creates a :file:`lib/pythonX.Y/site-packages` subdirectory
 (on Windows, this is :file:`Lib\site-packages`).
 If an existing directory is specified, it will be re-used.
 
-.. versionchanged:: 3.5
-   The use of ``venv`` is now recommended for creating virtual environments.
-
-.. deprecated-removed:: 3.6 3.8
-   :program:`pyvenv` was the recommended tool for creating virtual environments
-   for Python 3.3 and 3.4, and replaced in 3.5 by executing ``venv`` directly.
```

Comment:
Why was this removed?

Reply:
It is removed by mistake, I've added them again.